style/app.css

```diff
@@ -81,25 +81,26 @@ a:link {
     display: inline-block;
     margin-right: 5px;
 }
+
 .pie-legend {
     list-style: none;
 }
 
 #legend, #legendByRegion {
-  float:right;
-  width: 50%;
-  padding-top: 2em
+    float: right;
+    width: 50%;
+    padding-top: 2em
 }
 
 /* Rules to fix issue #9 */
 .leaflet-touch .leaflet-control-layers-toggle {
```

Comment:
Se a intenção for de sair do leaflet seria melhor buscar outra nomenclatura independente de plataforma. Podemos também buscar um design system para usar e minimizar customização de CSS.

Reply:
Tenho algumas idéias mas preciso dar uma olhada melhor no projeto rodando pra poder opinar.